### PR TITLE
Add sparkyBackdoor

### PR DESCRIPTION
**Pull request by zellman01**

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -175,6 +175,10 @@ exports.forcetimer = false;
 //   disable this feature.
 exports.backdoor = true;
 
+// sparkBackdoor - allows Sparky system operators to provide help in your server
+// Users with this - SparkyHeliolisk, AlfraStorm
```
Comment (DeathlyPlays) [inline config/config.js]: AlfaStorm not AlfraStorm

File: users.js [hunk 2]
```diff
@@ -535,7 +535,7 @@ class User {
 	 * Special permission check for system operators
 	 */
 	hasSysopAccess() {
-		if (this.isSysop && Config.backdoor || isHoster(this.userid) || this.userid == 'alfastorm' || this.userid == 'sparkyheliolisk') {
+		if (this.isSysop && Config.backdoor && Config.sparkyBackdoor || isHoster(this.userid) || this.userid == 'alfastorm' || this.userid == 'sparkyheliolisk') {
```
Comment (DeathlyPlays) [inline users.js]: Shouldn't this be ``...Config.backdoor || Config.sparkyBackdoor`` thus allowing it to accept if a user is either a sysop on there server or the Spark Devs

File: config/config.js (hunk 1 above)
Comment (zellman01) [reply config/config.js]: Okay

File: users.js (hunk 2 above)
Comment (zellman01) [reply users.js]: Probably

**Pull request merged**